ftw/keywordwidget/configure.zcml

```diff
@@ -83,4 +83,12 @@
         directory="./upgrades/select2js"
         />
 
+    <browser:page
+        for="z3c.form.interfaces.ISelectWidget"
```

Comment:
I think it would be nice to have a new keyword widget interface, which inherits from the `ISelectWidget`.